plugin/hooks.go

```diff
@@ -191,4 +193,20 @@ type Hooks interface {
 	//
 	// Minimum server version: 5.2
 	FileWillBeUploaded(c *Context, info *model.FileInfo, file io.Reader, output io.Writer) (*model.FileInfo, string)
+
+	// ReactionHasBeenAdded is invoked after the reaction has been committed to the database.
+	//
+	// Note that this method will be called for reactions added by plugins, including the plugin that
+	// added the reaction.
+	//
+	// Minimum server version: 5.29
```

Comment:
5.29 will be a quality release, so this should be moved to 5.30

Reply:
Actually, it's my understanding that v5.27 was the "last" of the tick-tock releases, and we're returning to monthly releases as of v5.28 and beyond.

Reply:
@hanzei - Happy to adjust to whichever :) Are we set on 5.30, or perhaps will be it 5.29 based on @lieut-data's comment?

Reply:
Heh, we /just/ pivoted to v5.29 being a quality release again, and eliminating tick-tock thereafter. So we can go with 5.30 now :D

Reply:
Updated to 5.30